src/test/kotlin/org/rust/ide/inspections/RsAssignToImmutableInspectionTest.kt

```diff
@@ -66,6 +70,7 @@ class RsAssignToImmutableInspectionTest : RsInspectionsTestBase(RsAssignToImmuta
         }
     """)
 
+    @ProjectDescriptor(WithStdlibRustProjectDescriptor::class)
```

Comment:
Do we really need stdlib in this test?

Reply:
@Undin Yes, because the type of `a[1]` is `<unknown>` otherwise

Reply:
Ok